python/lsst/ts/externalscripts/make_love_stress_tests.py

```diff
@@ -0,0 +1,318 @@
+                csc, salindex = salobj.name_to_name_index(name)
+                for stream in self.telemetry_streams[name]:
+                    await self.subscribe_to(csc, salindex, stream, "telemetry")
+            await self.handle_message_reception()
+
+
+class StressLOVE(salobj.BaseScript):
+    """Run a stress test for one or more CSCs.
+
+    Notes
+    -----
+    **Details**
+
+    * Run a LOVE stress test by generating several client connections
+    that will listen to every event and telemetry of the specified CSCs
+    """
+
+    def __init__(self, index):
+        super().__init__(index=index, descr="Run a stress test for one or more CSCs")
+
+        # number of clients to simulate on the stress test
+        self.number_of_clients = 10
+
+        # number of messages to store
+        # before calculating the latency
+        self.number_of_messages = 1000
+
+        # list to store clients connections,
+        # each one an instance of ManagerClient
+        self.clients = []
+
+        # list to store remote connections,
+        # each one an instance of salobj.Remote
+        self.remotes = []
+
+        # commands timeout
+        self.cmd_timeout = 10
+
+    @classmethod
+    def get_schema(cls):
+        schema_yaml = """
+            $schema: http://json-schema.org/draft-07/schema#
+            $id: https://github.com/lsst-ts/ts_standardscripts/StressLOVE.yaml
+            title: StressLOVE v1
+            description: Configuration for StressLOVE
+            type: object
+            properties:
+              location:
+                description: Domain of the running LOVE instance to stress
+                type: string
+              number_of_clients:
+                description: The number of clients to create
+                type: number
+              number_of_messages:
+                description: The number of messages to store before calculating the mean latency
+                type: number
+              data:
+                description: List of CSC_name[:index]
+                type: array
+                minItems: 1
+                items:
+                    type: string
+            required: [location, number_of_clients,data]
+            additionalProperties: false
+        """
+        return yaml.safe_load(schema_yaml)
+
+    def set_metadata(self, metadata):
+        """Compute estimated duration.
+
+        Parameters
+        ----------
+        metadata : SAPY_Script.Script_logevent_metadataC
+        """
+        # a crude estimate;
+        metadata.duration = len(self.clients) * 5
+
+    async def configure(self, config):
+        """Configure the script.
+
+        Specify the Stress test configurations:
+        - LOVE host location
+        - Number of clients
+        - Number of messages to store
+        - CSCs
+
+        Parameters
+        ----------
+        config : `types.SimpleNamespace`
+            Configuration with several attributes:
```

Comment:
I suggest pointing users to the get_schema method, instead of duplicating that info here (twice -- in this list and in the Notes section below). Duplicated information is hard to keep up to date.